Core/src/org/sleuthkit/autopsy/commandline/CommandLineIngestManager.java

```diff
@@ -0,0 +1,602 @@
+            }
+        }
+
+        public void run() {
+            LOGGER.log(Level.INFO, "Job processing task started");
+
+            try {
+                // read command line inputs
+                LOGGER.log(Level.INFO, "Autopsy is running from command line"); //NON-NLS
+                String dataSourcePath = "";
+                String baseCaseName = "";
+
+                // first look up all OptionProcessors and get input data from CommandLineOptionProcessor
+                Collection<? extends OptionProcessor> optionProcessors = Lookup.getDefault().lookupAll(OptionProcessor.class);
+                Iterator<? extends OptionProcessor> optionsIterator = optionProcessors.iterator();
+                while (optionsIterator.hasNext()) {
+                    // find CommandLineOptionProcessor
+                    OptionProcessor processor = optionsIterator.next();
+                    if (processor instanceof CommandLineOptionProcessor) {
+                        // check if we are running from command line
+                        dataSourcePath = ((CommandLineOptionProcessor) processor).getPathToDataSource();
+                        baseCaseName = ((CommandLineOptionProcessor) processor).getBaseCaseName();
+                    }
+                }
+
+                LOGGER.log(Level.INFO, "Data source path = {0}", dataSourcePath); //NON-NLS
+                LOGGER.log(Level.INFO, "Case name = {0}", baseCaseName); //NON-NLS
+                System.out.println("Data source path = " + dataSourcePath);
+                System.out.println("Case name = " + baseCaseName);
+
+                // verify inputs
+                if (dataSourcePath.isEmpty()) {
+                    LOGGER.log(Level.SEVERE, "Data source path not specified");
+                    System.out.println("Data source path not specified");
+                    return;
+                }
+
+                if (baseCaseName.isEmpty()) {
+                    LOGGER.log(Level.SEVERE, "Case name not specified");
+                    System.out.println("Case name not specified");
+                    return;
+                }
+
+                if (!(new File(dataSourcePath).exists())) {
+                    LOGGER.log(Level.SEVERE, "Data source file not found {0}", dataSourcePath);
+                    System.out.println("Data source file not found " + dataSourcePath);
+                    return;
+                }
+
+                // read options panel configuration
+                String rootOutputDir = UserPreferences.getCommandLineModeResultsFolder();
+                if (!(new File(rootOutputDir).exists())) {
+                    LOGGER.log(Level.SEVERE, "The output directory doesn't exist {0}", rootOutputDir);
+                    System.out.println("The output directory doesn't exist " + rootOutputDir);
+                    return;
+                }
+                rootOutputDirectory = Paths.get(rootOutputDir);
```

Comment:
Given that we write the case name and data source path to the log and to the console, shouldn't we also write the output directory path as well?